ArduPlane/quadplane.cpp

```diff
@@ -3532,6 +3550,21 @@ void QuadPlane::Log_Write_QControl_Tuning()
  */
 float QuadPlane::forward_throttle_pct()
 {
+    // handle special case where forward thrust motor is used instead of forward pitch.
+    // but not  in autotune as it will upset the results
+#if QAUTOTUNE_ENABLED
+    if (is_positive(q_fwd_thr_gain) && plane.control_mode != &plane.mode_qautotune) {
+#else
+    if (is_positive(q_fwd_thr_gain)) {
+#endif
+        calc_fwd_tilt_throttle();
+        // handle special case where we are using forward throttle instead of forward tilt in Q modes
+        return 100.0f * q_fwd_throttle;
+    } else {
+        q_fwd_throttle = 0.0f;
+        q_fwd_nav_pitch_lim_cd = -aparm.angle_max;
```

Comment:
This reset makes me nervous, what if we just stop calling `forward_throttle_pct` and get left with the 0 value in a mode that does not do forward throttle.

Reply:
The re-structuring should address this concern.